### PR TITLE
change in carousel

### PR DESCRIPTION
**Pull request by ManavRathod**
carousel me chnages kiye h dekhlena bhai

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -134,7 +135,7 @@
 		<section class="banner-section">
 			<div class="main-slider-carousel owl-carousel owl-theme">
 
-				<div class="slide" style="background-image: url(images/main-slider/image-1.jpg); width: 100%;">
+				<div class="slide" style="background-image: url(images/Samruddi-day-view1.jpg); width: 100%; height: 100vh;">
```
Comment (chintanshah9389) [inline index.html:138]: please use external css class rather then internal

**Review by chintanshah9389 — COMMENTED** [review]
please always use external css

**Review by Harsh-Rupesh-Shah — APPROVED** [review]

**Review by Harsh-Rupesh-Shah — APPROVED** [review]

**Comment by ManavRathod** [comment]
> please always use external css

tried to resolve but inline works better

**Comment by ManavRathod** [comment]
change was made in map, please review it

**Pull request merged**